src/views/About.vue

```diff
@@ -1,5 +1,60 @@
 <template>
   <div class="about">
-    <h1>This is under construction.</h1>
+    <SideBar class='sidebar'/>
+    <section class='about-content'>
+      <h2>Turing School of Software & Design Capstone Project</h2>
+      <p>The final project at Turing school is a full-stack venture to create an app
+        around an idea pitched by one of the students. Our goal was to develop a
+        crowdfunding dApp that made use of blockchain technology.
+      </p><p>
+        We had two weeks to
+        wireframe a front-end to interact with the blockchain and a back-end to supplement
+        it. We decided to use <b>Python</b> (+ ...other technologies) for our database and
+        <b>Vue</b> (+ ...other technologies) for our front-end. To interact with the blockchain,
+        the truffle framework/environment was used.<b>*None of these technologies were
+        explicitly taught at Turing*</b>
+      </p>
+      <h3>Why use blockchain?</h3>
+      <h3>Working wiht the blockchain</h3>
+      <h3>Back-End Technologies</h3>
+      <h3>Front-End Technologies</h3>
+    </section>
```

Comment:
Open to collaborate on content for this page.

Reply:
This is awesome Edwin, Ill definitely add to the moving forward.